packages/gmap-vue/examples/heatmap-layer.html

```diff
@@ -0,0 +1,39 @@
+<body>
+  <div id="root">
+    <gmap-map ref="mapRef" :zoom="7" :center="center" map-type-id="roadmap" style="width: 100%; height: 500px">
+        <gmap-marker v-for="(m, index) in markers" :key="index" :position="m.location" :clickable="true" :draggable="true" @click="center=m.location" />
+        <gmap-heatmap :data="markers" :options="{maxIntensity: 120, dissipating: false}" />
```

Comment:
```suggestion
        <gmap-heatmap-layer :data="markers" :options="{maxIntensity: 120, dissipating: false}" />
```
the name under which you registered the component was `GmapHeatmapLayer` because of that you need call them how `gmap-heatmap-layer` 😉